connectors/Telephony-Xtended-Service/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2445 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Telephony Xtended Serv Interf",
+    "description": "Integrate with BluIP's telephony system to take your call to the next level. Telephony Xtended Service Interface (XSI).",
+    "version": "1.0"

```

Comment:
@alainmenag Please add the x-ms-api-annotation in the info, Refer the swagger we have shared with you 

"x-ms-api-annotation": {
      "status": "Preview"
    }

Reply:
All done. I had a confusion of files. Thank you.